source/docs/software/examples-tutorials/drivesim-tutorial/updating-drivetrain-model.rst

```diff
@@ -41,6 +41,10 @@ There are three main steps to updating the model:
 
    .. code-tab:: c++
 
+      #include <frc/RobotController.h>
```

Comment:
Does it make sense to include this here? There are many other includes that are needed for the below code that are not included.

Reply:
We generally don't do the includes as it's a quick way to bloat up the documentation examples